cnames_active.js

```diff
@@ -2443,7 +2443,7 @@ var cnames_active = {
   "rebatov": "rebatov.github.io",
   "rebem": "rebem.github.io", // noCF? (don´t add this in a new PR)
   "rebornxp": "rebornxp.github.io/rebornxp",
-  "reciple": "falloutstudios.github.io/reciple-docs",
+  "reciple": "cname.vercel-dns.com",
```

Comment:
```suggestion
  "reciple": "cname.vercel-dns.com", // noCF
```
:wave: As the CI error notes, when using Vercel you'll want to disable the Cloudflare proxy for compatibility